2024/Open Source Summit North America/proposal.md

```diff
@@ -1,9 +1,24 @@
 # Session title
 
+How Should Your OSPO Work with InnerSource?
+
 # Description
 
+InnerSource is the application of open source development methodologies to company internal software development (https://innersourcecommons.org/).
+It was recently identified at the top Gartner's strategic trends for software engineering (https://gtnr.it/3tRYdVM)
+and is moving up the hype cycle (https://gtnr.it/4aX1vIa).
+With this attention on InnerSource, many organizations are creating formal programs to support InnerSource.
+How should an OSPO relate to a company's InnerSource program?
+We'll review approaches taken and lessons learned based on many company's experiences as shared in the TODO group and the InnerSource Commons.
+We'll summarize considerations to take into account when planning how to successfully navigate this relationship and opportunity in your own organization.
+
 # Benefits to the Ecosystem
 
+InnerSource is an emerging trend almost always overlaps with an OSPO when the practice arrives at an organization.
+If this situation hasn't happened yet at a given organization, then it is likely coming soon.
+A vanguard of organizations have worked out various approaches to managing this relationship.
+Reviewing this experience will prepare OSPO leaders in advance so that they are ready when this situation arrives in their organization.
```

Comment:
```suggestion
Reviewing this experience will prepare OSPO managers and other open source professionals working in, or alongside, OSPO-like entities (e.g., Open Source DevRels, compliance and security experts, project managers, etc.) in advance so that they are ready when this situation arrives in their organization.
```

Reply:
Yes!